### PR TITLE
log changes test

### PR DESCRIPTION
**Pull request by AntonyLeons**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/HMAN-658

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/integrationTest/java/uk/gov/hmcts/reform/hmc/repository/HearingResponseRepositoryIT.java
Comment (AntonyLeons) [inline src/integrationTest/java/uk/gov/hmcts/reform/hmc/repository/HearingResponseRepositoryIT.java:1]: fixes a flakey test, part of HMAN-601. but was annoying when testing

File: src/integrationTest/java/uk/gov/hmcts/reform/hmc/repository/HearingResponseRepositoryIT.java [hunk 1]
```diff
@@ -68,7 +69,8 @@ void testGetUnNotifiedHearingsWithStartDateTo() {
         assertNotNull(expected.getContent());
         assertEquals(2, expected.getContent().size());
         assertEquals(2, expected.getTotalElements());
-        assertEquals(expectedHearingIds, expected.getContent());
+        List<Long> actualHearingIds = expected.getContent().stream().sorted().collect(Collectors.toList());;
```
Comment (markdathornehmcts) [inline src/integrationTest/java/uk/gov/hmcts/reform/hmc/repository/HearingResponseRepositoryIT.java:72]: Interesting. I resolved in another PR using new HashSet for both expected and actual Lists.

**Review by markdathornehmcts — APPROVED** [review]
LGTM

File: src/integrationTest/java/uk/gov/hmcts/reform/hmc/repository/HearingResponseRepositoryIT.java (hunk 1 above)
Comment (AntonyLeons) [reply src/integrationTest/java/uk/gov/hmcts/reform/hmc/repository/HearingResponseRepositoryIT.java:72]: https://build.hmcts.net/job/HMCTS_d_to_i/job/hmc-cft-hearing-service/job/PR-474/34/pipeline-console/?selected-node=86
idk, might still be broken but hasn't failed yet.

**Pull request merged**